integration/js/script/run-test

```diff
@@ -77,67 +77,116 @@ cd ../
-   content_share_test_suite_one)
-      r content_share_screen_capture_test.config.json
-      r meeting_leave_content_share_test.config.json
-      r content_share_video_test.config.json
-      ;;
-   content_share_test_suite_two)
-      r content_share_only_allow_two_test.config.json
-      r content_share_join_later_test.config.json
-      r app_quit_content_share_test.config.json
-      ;;
-   data_message)
-      r data_message_test.config.json
-      ;;
-   meeting_readiness_checker)
-      r meeting_readiness_checker_test.config.json
-      ;;
-   messaging)
-      r messaging_session_test.config.json
-      ;;
-   transcription)
-      r transcription_test.config.json
-      ;;
-   background_blur)
-      r background_blur_test.config.json
-      ;;
-   echo_reduction)
-      r echo_reduction_test.config.json
+   integration)
+   case $2 in
+      audio)
+         r configs/integrationTest/audio_test.config.json
+         r configs/integrationTest/app_quit_audio_test.config.json
+         r configs/integrationTest/meeting_leave_audio_test.config.json
+         ;;
+      video)
+         r configs/integrationTest/video_test.config.json
+         r configs/integrationTest/video_test_processor.config.json
+         #  TODO: Re-enable simulcast test after it has been fixed
+         # r configs/integrationTest/video_test_simulcast.config.json
+         r configs/integrationTest/app_quit_video_test.config.json
+         r configs/integrationTest/meeting_leave_video_test.config.json
+         r configs/integrationTest/reconnection_test.config.json
+         ;;
+      video_test_app)
+         r configs/integrationTest/test_app_video_test.config.json
+         r configs/integrationTest/test_app_video_pin_test.config.json
+         ;;
+      meeting_end)
+         r configs/integrationTest/meeting_end_test.config.json
+         ;;
+      content_share_test_suite_one)
+         r configs/integrationTest/content_share_screen_capture_test.config.json
+         r configs/integrationTest/meeting_leave_content_share_test.config.json
+         r configs/integrationTest/content_share_video_test.config.json
+         ;;
+      content_share_test_suite_two)
+         r configs/integrationTest/content_share_only_allow_two_test.config.json
+         r configs/integrationTest/content_share_join_later_test.config.json
+         r configs/integrationTest/app_quit_content_share_test.config.json
+         ;;
+      data_message)
+         r configs/integrationTest/data_message_test.config.json
+         ;;
+      meeting_readiness_checker)
+         r configs/integrationTest/meeting_readiness_checker_test.config.json
+         ;;
+      messaging)
+         r configs/integrationTest/messaging_session_test.config.json
+         ;;
+      transcription)
+         r configs/integrationTest/transcription_test.config.json
+         ;;
+      background_blur)
+         r configs/integrationTest/background_blur_test.config.json
+         ;;
+      echo_reduction)
```

Comment:
Do we want to drop this and re-add later after launch?

Reply:
We can let it be for now. The `echo_reduction` tests are currently not enabled. When someone will open a PR this will not run and I am not adding it to browser compatibility test as of now.